executor/pt_executor.cpp

```diff
@@ -31,10 +31,10 @@ PTExecutor::PTExecutor(
     bool record_stdout_and_err
 ) : ProxyExecutor ( proxy_path, std::vector<std::string>(), argv, exec_timelimit_ms, exec_memlimit, forksrv,
                     path_to_write_input, 0, 0, record_stdout_and_err ),
-    path_shm_size ( PTExecutor::PATH_SHM_SIZE ),
-    fav_shm_size ( PTExecutor::FAV_SHM_SIZE )
+    AFLPTPathCoverage(PTExecutor::PATH_SHM_SIZE),
+    AFLPTPathFav(PTExecutor::FAV_SHM_SIZE)
 {
-    if (path_shm_size > 0 || fav_shm_size > 0) {
+    if (AFLPTPathCoverage::GetMapSize() > 0 || AFLPTPathFav::GetMapSize() > 0) {
         has_shared_memories = true;
     }
```

Comment:
It seems `has_shared_memories` is no longer used in `PTExecutor`. This is not so related to this PR, but probably we can remove it.

Reply:
Oops, sorry. PTExecutor inherits from ProxyExecutor.